### DISCUSSION TITLE
Ability to choose which can be seen in private profiles

### DISCUSSION THREAD

**Discussion by ZeroAurora** [community/community#16752] [Other Feature Feedback, Questions, & Ideas]
For example:
I may want to display to others my commits only, not my issues.
I may want to hide my contributions, but the pinned repos are still visible.
Yes that's it 🤔 Hiding everything is good, but far from perfect

**Comment by handipriyono** [comment]
agreed

**Comment by Nickishero** [comment]
I agree.
It says "contributed to" and names the public repositories i contributed to on my profile, but I would like to hide one (or more, or all) of these. However, I **do** want to show my commits.

The reason why I don't want these to show on my profile, is because some of these contain private information. I want to keep this git account private; i dont want to be doxxed so easily.

**TL;DR**: Basically allow the opportunity to be a 'private' contributor in public repo's on your profile. They won't be shown on your profile.
